lightly/loss/tico_loss.py

```diff
@@ -31,31 +47,39 @@ class TiCoLoss(torch.nn.Module):
 
     def __init__(
         self,
-        beta_param: float = 0.9,
-        ro_param: float = 20.0,
-        gather_distributed : bool = False,
+        beta: float = 0.9,
+        rho: float = 20.0,
+        C: torch.Tensor = Variable(torch.zeros(256, 256), requires_grad=True),
```

Comment:
Actually, letting C as an argument might be useful because if someone wants to change the last layer dimension can overwrite C as Variable(torch.zeros(32, 32), requires_grad=True) for example.

Reply:
My proposal below would do this automatically: https://github.com/lightly-ai/lightly/pull/1016#discussion_r1055286323